### PR TITLE
First Pushrequest - Proofreading

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Please check if it's correct that way.
For some reason it says:  "4,190 additions and 4,235 deletions."

### CODE REVIEW

File: doc/usermanual/po/de.po [hunk 1]
```diff
@@ -715,7 +716,8 @@ msgstr ""
 "Libraries, die zur Laufzeit geladen werden. Mit dieser Option können Sie "
 "darktable mitteilen, wo es nach diesen Bibliotheken suchen soll. Die "
 "Voreinstellung hängt von Ihrer Installation ab; typische Orte sind <quote>/"
-"opt/darktable/lib/darktable/</quote> und <quote>/usr/lib/darktable/</quote>."
+"opt/darktable/lib/darktable/</quote> und <quote>/usr/lib/darktable/</quote>. "
+"# Hier ist ein anderer Pfad angegeben als im englischen Original."
```
Comment (mepi0011) [inline doc/usermanual/po/de.po]: Path is changed according to the English version.

Please do not insert comments directly into the translation. Instead mark the translation as not finished and insert a comment via poedit (comment button)

File: doc/usermanual/po/de.po [hunk 2]
```diff
@@ -618,7 +618,8 @@ msgstr ""
 "als aktuelle Sammlung. Wenn darktable bereits läuft (und die gleiche "
 "Datenbank benutzt) dann werden die Bilder und Ordner in der bestehenden "
 "Instanz geöffnet, wobei D-Bus zur Kommunikation zwischen den Prozessen "
-"verwandt wird."
+"verwandt wird. # Hier wird mehr übersetzt als im englischen Quelltext da "
+"ist. Ist das so gewollt?"
```
Comment (mepi0011) [inline doc/usermanual/po/de.po]: Translation changed according to English version.

**Comment by mepi0011** [comment]
Please remember to update your fork and your local version! The command git status should help you. It must be something like git fetch or git pull.

**Comment by Hofer-Julian** [comment]
Okay. I will do [that](https://help.github.com/articles/syncing-a-fork/) regularly, if that is what you meant.

Is it necessary to build the whole darktable-project afterwards, or is "make update-usermanual-de" enough?

Also: Should I open a new pullrequest for the next commits or will they also be added to this one?

**Comment by mepi0011** [comment]
Hi,

Syncing a fork is what I meant.

'make update-usermanual-de' should enough

If I accept a pull request, a new pull request is needed for further changes.
I will always try to react as soon as possible to open pull requests and process them.

**Pull request merged**